modules/ding_user/ding_user.module

```diff
@@ -682,6 +682,32 @@ function ding_user_default_authname($name) {
 }
 
 /**
+ * Private key variable used to generate tokens.
+ *
+ * Apparently the key generated in drupal_get_private_key() may change,
+ * so we'll get it from 'ding_private_key' variable instead.
+ * Preferably set the 'ding_private_key' variable in local.settings.php
```

Comment:
A reference to the issue on drupal.org would be good here.

Reply:
@kasperg: I can't seem to find any drupal.org issues regarding it - do you have a link?

Reply:
:+1: You have added it.